ar_blocks/Structure goal_structure
---
ar_blocks/Structure final_structure
---
ar_blocks/Structure updated_structure 
